site/_posts/2019-11-18-cis-benchmark-plugin.md

```diff
@@ -94,9 +94,11 @@ We can chain these desired commands together as a single bash command and then s
 $ sonobuoy gen plugin \
     --name kube-bench-worker \
     --image=schnake/kube-bench:v0.2.0-demo \
-    --cmd=/bin/sh
-    --args=-c
-    --args=”kube-bench --version 1.13 --outputfile /tmp/results/output.xml --junit ; echo -n /tmp/results/output.xml > /tmp/results/done” \
+    --type=DaemonSet \
```

Comment:
The type should also be DaemonSet, right?

Reply:
Yes; thanks! Sorry these were such a mess; it was a mix of manually making the files and then writing the commands afterwards.